### ISSUE TITLE
No "buildTarget" for buildPhase

### ISSUE THREAD

**Issue by ledettwy** [NixOS/nixpkgs#28047]
## Issue description

Build phases like installPhase and checkPhase have various "target" variables which control the make target invoked during the phase.  But for some reason buildPhase does not.  Can this be added?

### Workaround

It looks like the current workaround is to use buildFlags instead (GNU HURD derivation does this: https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/gnu/hurd/default.nix#L82)

## Technical details

* Nixpkgs version: 17.03.1556.4d1e1d07f9

**Comment by aanderse** [comment]
Is this "fixed"? Not listed in the nixpkgs manual, but I see buildTargets used several places so I assume it exists?

https://github.com/NixOS/nixpkgs/search?q=buildTargets&unscoped_q=buildTargets

**Comment by veprbl** [comment]
@aanderse No, this isn't fixed, but indeed using buildFlags should be enough:
https://github.com/NixOS/nixpkgs/blob/990eaa30d9b93e56a21d4f91e0421d4fe6eacafb/pkgs/stdenv/generic/setup.sh#L1025-L1033
https://github.com/NixOS/nixpkgs/blob/990eaa30d9b93e56a21d4f91e0421d4fe6eacafb/pkgs/stdenv/generic/setup.sh#L1064-L1073

**Comment by ShamrockLee** [comment]
IMO, having `buildTargets` would make the interface more consistent and ensure that the targets appear at the tail of the argument list.
